dali/base/dadfs.cpp

```diff
@@ -13454,14 +13454,14 @@ IPropertyTreeIterator *deserializeFileAttrIterator(MemoryBuffer& mb, unsigned nu
             else
                 sizeDiskSize = file->getPropInt64(getDFUQResultFieldName(DFUQRForigsize), 0);
             double sizeGB = sizeDiskSize / ((double)1024 * 1024 * 1024);
-            cost_type atRestCost = money2cost_type(calcFileAtRestCost(nodeGroup, sizeGB, fileAgeDays));
-            file->setPropInt64(getDFUQResultFieldName(DFUQRFatRestCost), atRestCost);
+            cost_type atRestCostType = calcFileAtRestCost(nodeGroup, sizeGB, fileAgeDays);
```

Comment:
trivial: don't think this variable name needs/is helped by having suffix "Type", and similar below.